modules/common/logging/logs-source.nix

```
@@ -0,0 +1,46 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  hostName,
+  ...
+}: let
+  admin-ip-port = "admin-vm-debug:${toString config.ghaf.logging.listener.port}";
+  cfg = config.ghaf.logging.client;
+in {
+  config = lib.mkIf cfg.enable {
+    environment.etc."alloy/config.alloy" = {
+      text = ''
+        discovery.relabel "journal" {
+          targets = []
+          rule {
+            source_labels = ["__journal__systemd_unit"]
+            target_label  = "unit"
+          }
```

```suggestion
      rule {
        source_labels = ["__journal__systemd_unit"]
        target_label = "systemd_unit"
      }
      rule {
        source_labels = ["__journal__hostname"]
        target_label = "nodename"
      }
      rule {
        source_labels = ["__journal_syslog_identifier"]
        target_label = "syslog_identifier"
      }

```

(maybe different indent ofc)